mllib/src/test/scala/org/apache/spark/ml/classification/ProbabilisticClassifierSuite.scala

```diff
@@ -56,6 +56,21 @@ class ProbabilisticClassifierSuite extends SparkFunSuite {
     val testModel = new TestProbabilisticClassificationModel("myuid", 2, 2)
     assert(testModel.friendlyPredict(Vectors.dense(Array(1.0, 2.0))) === 1.0)
   }
+
+  test("test tiebreak") {
+    val testModel = new TestProbabilisticClassificationModel("myuid", 2, 2)
+      .setThresholds(Array(0.4, 0.4))
+    assert(testModel.friendlyPredict(Vectors.dense(Array(0.6, 0.6))) === 0.0)
+  }
+
+  test("bad thresholds") {
+    intercept[IllegalArgumentException] {
```

Comment:
No they don't seem to be probabilities. In fact I don't see a theoretical reason why the sum <=1, since they're just scalings or "weights". Requiring all > 0 of course makes practical sense. Anyway we should just match what they do as that is what this `thresholds` implementation was based on.